docs/pull_request_template.md

```diff
@@ -0,0 +1,59 @@
+Please make sure you have read the PR guidelines below, and checked all boxes that apply at the bottom.
+
+## Changes proposed in this pull request
+
+**Please describe here the main features / changes proposed for review and integration in fMRIPrep**
+
+If this PR addresses some existing problem, please use GitHub's citing tools (eg. ``ref #``, ``closes #`` or ``fixes #``).
```

Comment:
Right now the contributing guide asks for people to make sure there's an issue open before starting a pull request, so I think it's safe to ask them to reference an existing issue. Unless that's something you'd like to change in the guide !

Reply:
Well, it exactly says that if the issue does not exist you may open it up (to avoid overlapping work, etc.). So I'll take that into account.